flow/model/qvalue/avro_converter.go

```diff
@@ -323,6 +323,9 @@ func (c *QValueAvroConverter) processNumeric() (interface{}, error) {
 		return nil, fmt.Errorf("invalid Numeric value: expected *big.Rat, got %T", c.Value.Value)
 	}
 
+	scale := 9
```

Comment:
This should be a constant somewhere else, used in same places we use 9 for scale elsewhere